fuzz/fuzz_targets/block.rs

```diff
@@ -132,10 +135,37 @@ fn memfd_create(name: &ffi::CStr, flags: u32) -> Result<RawFd, io::Error> {
 pub struct NoopVirtioInterrupt {}
 
 impl VirtioInterrupt for NoopVirtioInterrupt {
-    fn trigger(
-        &self,
-        _int_type: VirtioInterruptType,
-    ) -> std::result::Result<(), std::io::Error> {
+    fn trigger(&self, _int_type: VirtioInterruptType) -> std::result::Result<(), std::io::Error> {
         Ok(())
     }
 }
+
+fn wait_queue_event_processed(queue_evt: EventFd) {
+    let monitor_evt = EventFd::new(libc::EFD_NONBLOCK).unwrap();
+    monitor_evt.write(1).unwrap();
```

Comment:
Why is the monitor event needed in the first place? And why are you writing to it? It seems the following code will directly exit the loop since it's going to find one event pending (the monitor event that you've just triggered).

Reply:
Please see comment above. Please let me know if that makes sense.